torchao/__init__.py

```diff
@@ -14,6 +8,13 @@
     from . import _C
```

Comment:
Need to import `_C` first since `to/from_float6_e3m2()` (from `dtypes`) calls C++ extension for CPU.